From 678d383d57172348589d20e1958849adb540cdfd Mon Sep 17 00:00:00 2001
Message-Id: <678d383d57172348589d20e1958849adb540cdfd.1426664743.git.chang-joon.lee@intel.com>
In-Reply-To: <6cf4814be6b971f0e7141f9bb9a3acd992d576bf.1426664743.git.chang-joon.lee@intel.com>
References: <6cf4814be6b971f0e7141f9bb9a3acd992d576bf.1426664743.git.chang-joon.lee@intel.com>
From: Uma Shankar <uma.shankar@intel.com>
Date: Sat, 28 Feb 2015 00:41:11 +0530
Subject: [PATCH 3/3] FOR_UPSTREAM [VPG]: drm/i915: Removed redundant delay in
 mipi sequence

There is an extra/duplicate panel power delay being
programmed in the MIPI enabling sequence. This is
delaying display resume time. Removing the same.

Issue: IRDA-3305
Change-Id: I83468f8bc53b6ed48a4fd751a81579c3cc16f23e
Signed-off-by: Uma Shankar <uma.shankar@intel.com>
---
 drivers/gpu/drm/i915/intel_dsi.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_dsi.c b/drivers/gpu/drm/i915/intel_dsi.c
index 33a5e8c..9ff2b97 100644
--- a/drivers/gpu/drm/i915/intel_dsi.c
+++ b/drivers/gpu/drm/i915/intel_dsi.c
@@ -373,8 +373,6 @@ static void intel_dsi_pre_enable(struct intel_encoder *encoder)
 	/* put device in ready state */
 	intel_dsi_device_ready(encoder);
 
-	msleep(intel_dsi->panel_on_delay);
-
 	if (intel_dsi->dev.dev_ops->panel_reset)
 		intel_dsi->dev.dev_ops->panel_reset(&intel_dsi->dev);
 
-- 
1.7.9.5

